data/authors/oleg-zinovyev.json

```
@@ -0,0 +1,8 @@
+{
+  "name": "Oleg Zinovyev",
+  "bio": "Technical and content writer at JetBrains",
+  "social": [
+    { "linkedin": "https://www.linkedin.com/in/oleg-znv" }
```

Please don't forget the comma. Hugo cannot build the website because of it :)

```suggestion
    { "linkedin": "https://www.linkedin.com/in/oleg-znv" },
```

---

Oh, sorry 
✅